projects/ngneat/forms-manager/src/lib/forms-manager.ts

```diff
@@ -389,6 +391,25 @@ export class NgFormsManager<FormsState = any> {
     return this;
   }
 
+  private getValueChangeStreams(control: AbstractControl) {
+    const streams = [];
+
+    if (control.updateOn === 'blur') {
```

Comment:
Here we check if the main FormGroup is set to updateOn=blur then we don't need to check for the rest of controls inside in the FormGroup, a case like this:
`stepOne =  this.fb.group({
    textarea: [''],
    skills: new FormArray([new FormControl('JS')]),
  },{updateOn:'blur'});`